### PR TITLE
Release 7.10.2.0

### PR DESCRIPTION
**Pull request by baptabl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
+| 7.0.1 | 7.0.1.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.0.1.0/geopoint-clustering-aggregation-7.0.1.0.zip|
+| 7.1.1 | 7.1.1.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.1.1.0/geopoint-clustering-aggregation-7.1.1.0.zip|
+| 7.2.0 | 7.2.0.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.2.0.0/geopoint-clustering-aggregation-7.2.0.0.zip|
+| 7.4.0 | 7.4.0.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.4.0.0/geopoint-clustering-aggregation-7.4.0.0.zip|
+| 7.5.1 | 7.5.1.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.5.1.0/geopoint-clustering-aggregation-7.5.1.0.zip|
+| 7.6.0 | 7.6.0.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.6.0.0/geopoint-clustering-aggregation-7.6.0.0.zip|
+| 7.7.0 | 7.7.0.0 | https://github.com/opendatasoft/elasticsearch-aggregation-geoclustering/releases/download/v7.7.0.0/geopoint-clustering-aggregation-7.7.0.0.zip|
```
Comment (benj-moreau) [inline README.md:40]: add v7.10.2.0

**Review by benj-moreau — APPROVED** [review]
Tested with jdk 15:
- tested the 25 zoom values
- tested all params 
- with and without cache
All results are ok 👍

File: README.md [hunk 2]
```diff
@@ -18,76 +58,141 @@ Then it merges these cells if the distance between two clusters' centers is lowe
 }
 ```
 Input parameters :
- - `field`: must be of type geo_point.
- - `zoom`: mandatory integer parameter between 0 and 20. It represents the zoom level used in the request to aggregate geo points.
- - `radius`: radius in pixel. It is used to compute a geohash precision then to merge cluters based on this distance. Default to `50`. 
- - `ratio`: ratio used to make a second merging pass. If the value is `0`, no second pass is made. Default to `2`. 
- - `extent`: Extent of the tiles. Default to `256`
+- `field`: must be of type [geo_point](https://www.elastic.co/guide/en/elasticsearch/reference/7.10/geo-point.html)
+- `zoom`: mandatory integer parameter between 0 and 25. It represents the zoom level used in the request to aggregate geo points
```
Comment (mhugo) [inline README.md:62]: I guess the level refers to the TMS zoom level ? https://docs.nearmap.com/pages/viewpage.action?pageId=11206698

File: README.md [hunk 3]
```diff
@@ -18,76 +58,141 @@ Then it merges these cells if the distance between two clusters' centers is lowe
...
+- `radius`: radius in pixel. It is used during the reduce phase to merge close clusters. Default to `40`
```
Comment (mhugo) [inline README.md:63]: Which unit for the radius ? meters ?

File: README.md [hunk 4]
```diff
@@ -18,76 +58,141 @@ Then it merges these cells if the distance between two clusters' centers is lowe
...
+- `ratio`: ratio used to make a second merging pass during the reduce phase. If the value is `0`, no second pass is made. Default to `0`
```
Comment (mhugo) [inline README.md:64]: Ratio of what over what ?

File: README.md [hunk 5]
```diff
@@ -18,76 +58,141 @@ Then it merges these cells if the distance between two clusters' centers is lowe
...
+- `extent`: Extent of the tiles. Default to `256`
```
Comment (mhugo) [inline README.md:65]: Why do we need to give a tile extent ? And which unit it is ? pixels I guess ?

File: README.md (hunk 3 above)
Comment (baptabl) [reply README.md:63]: pixels

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (baptabl) [reply README.md:64]: I guess it should be named `second_pass`. Do you think we should rename it?

File: README.md (hunk 5 above)
Comment (baptabl) [reply README.md:65]: Yes. We never used it anywhere, but maybe some people do, so let's keeps it (?)

**Pull request merged**